src/phoenix/server/api/types/Span.py

```diff
@@ -169,6 +179,20 @@ def _extract_attributes(row: "Series[Any]") -> "Series[Any]":
     )
 
 
+def to_gql_span_events(row: "Series[Any]") -> List[SpanEvent]:
+    """
+    Converts a dataframe row to a list of graphQL span events
+    """
+    return [
+        SpanEvent(
+            name=event["name"],
+            message=event["message"],
+            timestamp=datetime.fromisoformat(event["timestamp"]),
+        )
+        for event in json.loads(row)
+    ]
```

Comment:
this function can also be a class method: `SpanEvent.from_mapping`
```suggestion
def to_gql_span_event(event: Mapping[str, Any]) -> SpanEvent:
    return SpanEvent(
        name=event["name"],
        message=event["message"],
        timestamp=datetime.fromisoformat(event["timestamp"]),
    )
```

Reply:
Done